### PR TITLE
IntEum for python

### PR DESCRIPTION
**Pull request by waszil**
Implemented `enum.IntEnum` representation for enums for python. Implements #40 .

### CODE REVIEW

**Comment by waszil** [comment]
plus, I have added .bitproto file syntax highlighting support for PyCharm as well.

File: compiler/bitproto/renderer/impls/py/renderer.py [hunk 1]
```diff
@@ -289,6 +344,20 @@ def before(self) -> None:
...
+        pass
+
+    @override(BlockWrapper)
+    def before(self) -> None:
+        self.push("@staticmethod")
+        self.push("def dict_factory(kv_pairs):")
+        self.push(
+            f"    return {{k: v for k, v in kv_pairs if not k.startswith('{_enum_field_proxy_prefix}')}}"
+        )
+        self.push_definition_docstring(indent=4)
```
Comment (hit9) [inline compiler/bitproto/renderer/impls/py/renderer.py:358]: >         self.push_definition_docstring(indent=4)

Hmm, should this line be removed?

**Comment by hit9** [comment]
Hi, this pr looks great, it also takes into account backward compatibility.

LGTM.

**Review by hit9 — APPROVED** [review]

File: compiler/bitproto/renderer/impls/py/renderer.py (hunk 1 above)
Comment (waszil) [reply compiler/bitproto/renderer/impls/py/renderer.py]: Oh, yes, indeed, typical copy-paste artifact... I've removed it and pushed the commit to the branch.

**Comment by hit9** [comment]
https://github.com/hit9/bitproto/releases/tag/v0.4.5

**Pull request merged**